src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>MetricServer for the Prometheus.Client</Description>
+    <Copyright>2019 © Sergey Kuznetsov</Copyright>
+    <AssemblyTitle>Prometheus.Client.MetricServer</AssemblyTitle>
+    <VersionPrefix>3.0.2</VersionPrefix>
+    <Authors>Sergey Kuznetsov</Authors>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
```

Comment:
We should add neststandard2.0 and netcoreapp2.2 for Kestrel

Reply:
netstandard2.0 was added.